contrib/examples/actions/workflows/mistral-env-var.yaml

```diff
@@ -0,0 +1,14 @@
+version: '2.0'
+
+examples.mistral-env-var:
+    description: A basic workflow that illustrates how to get the workflow's env vars.
+    type: direct
+    output:
+        stdout: <% $.stdout %>
+    tasks:
+        task1:
+            action: core.local
+            input:
+                cmd: echo <% env().st2_execution_id %>
+            publish:
+                stdout: <% $.task1.stdout %>
```

Comment:
I would also publish env to make it ready for an exploring user to see what else good inside env. 
`env: <% env() %> `